ibis/pyspark/compiler.py

```diff
@@ -220,53 +222,119 @@ def fn(col):
 
 @compiles(ops.Any)
 def compile_any(t, expr, scope, context=None, **kwargs):
-    return compile_aggregator(t, expr, scope, F.max, context)
+
+    def fn(col):
+        if "window" in kwargs:
```

Comment:
I think the default for string used in ibis is single quotes. probably we should keep all string using single quotes.

Reply:
Done